docs/hugo/content/architecture-decision-records/ADR-2022-02-Adoption-Policy.md

```diff
@@ -0,0 +1,127 @@
+# Adoption Policy 
+
+## Context
+
+Resources which exist in Azure prior to being created in Kubernetes are "adopted" by the operator when the resource
+is created in Kubernetes. Today this adoption process is automatic and implicit. As a user there isn't an easy way
+to tell if a resource you're about to `kubectl apply` already exists in Azure (will be adopted) or does not exist in
+Azure (will be created).
+
+We have recently proposed support for [reconcile-policy](https://github.com/Azure/azure-service-operator/pull/2060) which
+is related to adoption. Some of `reconcile-policy`'s primary use cases are for when the user does not want ASO to
+fully manage an adopted resource.
+
+The following are principles we should follow related to resource adoption:
+
+1. **The operator should adopt resources by default:** This is standard goal-seeking/desired-state behavior. 
+   If you create a resource in Kubernetes that says: "I expect there to be a Virtual Machine that looks like this
+   at location X" and that Virtual Machine already exists, then we are in the desired state.
+   The alternative would be to error if the resource already exists, which goes against the Kubernetes model.
+2. **Users must be able to instruct the operator to stop making changes to the backing Azure resource:**
```

Comment:
Suggest a blank line between each principle, since they're paragraphs.

```suggestion
   The alternative would be to error if the resource already exists, which goes against the Kubernetes model.
   
2. **Users must be able to instruct the operator to stop making changes to the backing Azure resource:**
```